packages/text-annotator/src/SelectionHandler.ts

```diff
@@ -135,7 +135,11 @@ export const SelectionHandler = (
     const clickSelect = () => {
       const { x, y } = container.getBoundingClientRect();
 
-      const hovered = store.getAt(evt.clientX - x, evt.clientY - y, currentFilter);
+      const hovered =
+        evt.target instanceof Node &&
+        container.contains(evt.target) &&
```

Comment:
Added the `container.contains` as a small optimization that will immediately dismiss clicks outside of the `container`. So we won't need to execute the relatively heavier `getAt` method